test/pack/universal-generator/universalGenerator.test.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright © 2019 Atomist, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    GitHubRepoRef,
+    ProjectFile,
+} from "@atomist/automation-client";
+import { GeneratorRegistration } from "@atomist/sdm";
+import * as assert from "assert";
+import { UniversalTransform } from "../../../lib/pack/universal-generator/generatorSupport";
+import {
+    AssertGeneratorResult,
+    assertUniversalGenerator,
+} from "../../../lib/pack/universal-generator/test/assertGenerator";
+
+export const SpringGeneratorRegistration: GeneratorRegistration<{ name: string }> = {
```

Comment:
Does this need to be exported? If not, remove it. If it does, it should be moved to a file without tests, otherwise the tests in the file will be run additional times for each time it is imported.